distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java

```diff
@@ -13,69 +13,101 @@
  */
 package org.codice.ddf.itests.common.config;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.with;
+
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.Dictionary;
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 import org.codice.ddf.configuration.DictionaryMap;
 import org.codice.ddf.itests.common.AdminConfig;
 import org.osgi.service.cm.Configuration;
 
 public class ConfigureTestCommons {
 
-  public static final String METACARD_VALIDITITY_FILTER_PLUGIN_SERVICE_PID =
+  private static final String METACARD_VALIDITY_FILTER_PLUGIN_SERVICE_PID =
       "ddf.catalog.metacard.validation.MetacardValidityFilterPlugin";
 
-  public static final String METACARD_VALIDITITY_MARKER_PLUGIN_SERVICE_PID =
+  private static final String METACARD_VALIDITY_MARKER_PLUGIN_SERVICE_PID =
       "ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin";
 
-  public static final String METACARD_ATTRIBUTE_SECURITY_POLICY_PLUGIN_PID =
+  private static final String METACARD_ATTRIBUTE_SECURITY_POLICY_PLUGIN_PID =
       "org.codice.ddf.catalog.security.policy.metacard.MetacardAttributeSecurityPolicyPlugin";
 
-  public static final String AUTH_Z_REALM_PID = "ddf.security.pdp.realm.AuthzRealm";
+  private static final String AUTH_Z_REALM_PID = "ddf.security.pdp.realm.AuthzRealm";
 
   private ConfigureTestCommons() {}
 
-  public static void configureMetacardValidityFilterPlugin(
-      List<String> securityAttributeMappings, AdminConfig configAdmin) throws IOException {
-    Configuration config =
-        configAdmin.getConfiguration(METACARD_VALIDITITY_FILTER_PLUGIN_SERVICE_PID, null);
-    Dictionary<String, Object> properties = new DictionaryMap<>();
-    properties.put("attributeMap", securityAttributeMappings);
-    config.update(properties);
+  public static Dictionary<String, Object> configureService(
```

Comment:
❓  We have a few places that already configure a service and wait for it to be updated. Any reason not to use the service manager or sync installer instead? 

https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java#L127  

https://github.com/codice/ddf/blob/829ff3a02a659d499b684a0a475255caebec5e1c/platform/sync-installer/sync-installer-impl/src/main/java/org/codice/ddf/sync/installer/impl/SynchronizedInstallerImpl.java#L173